yoti-sdk-api/src/test/java/com/yoti/api/client/HumanProfileTest.java

```diff
@@ -374,6 +376,43 @@ public void getIdentityProfileReport_shouldReturnValueForIdentityProfileReport()
         assertThat(value.get("verification_report"), is(notNullValue()));
     }
 
+    @Test
+    public void getAttributeById_shouldReturnAttributeById() {
+        String anId = "anId";
+        Attribute<JpegAttributeValue> imageAttribute = createImageAttribute(anId);
+
+        List<Attribute<?>> attributes = new ArrayList<>();
+        attributes.add(imageAttribute);
+        attributes.add(createImageAttribute("anotherId"));
+
+        HumanProfile humanProfile = new HumanProfile(attributes);
+
+        Attribute<?> attribute = humanProfile.getAttributeById(anId);
+
+        assertThat(attribute, is(imageAttribute));
+    }
+
+    @Test
+    public void getAttributeById_shouldReturnNull() {
+        List<Attribute<?>> attributes = new ArrayList<>();
+        attributes.add(createImageAttribute("anId"));
+        attributes.add(createImageAttribute("anotherId"));
+
+        HumanProfile humanProfile = new HumanProfile(attributes);
+
+        Attribute<?> attribute = humanProfile.getAttributeById("unknown_id");
+
+        assertNull(attribute);
+    }
+
+    private static Attribute<JpegAttributeValue> createImageAttribute(String id) {
+        return new Attribute<>(
+                id,
+                String.format("name_%s", id),
```

Comment:
Could add a test for adding two attributes with the same name? This is what will be returned on the profile, so we will need to test for that 👍

Reply:
sure